addons/udes_stock/models/stock_location.py

```diff
@@ -853,6 +853,13 @@ def update_scanning_date_start(self):
         self.ensure_one()
         return None
 
+class Route(models.Model):
```

Comment:
Can we put this in its own file so we have one model per file, please?

Do you mind moving Orderpoint into its own file too please?

Thanks